docs/content/manual/pre-release/ha/single-replica-node-down.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Single replica node down
+---
+
+## Related Issues
+https://github.com/longhorn/longhorn/issues/2329
+https://github.com/longhorn/longhorn/issues/2309
+
+## Default Setting
+`Automatic salvage` is enabled.
+
+## Node restart/down scenario
```

Comment:
In this case, I assume this is `Pod Delete Policy When Node is Down` disabled/do-nothing?

Reply:
Thanks, fixed.